sky/clouds/aws.py

```diff
@@ -127,9 +127,27 @@ def get_default_ami(cls, region_name: str, instance_type: str) -> str:
         amis = {
             'us-east-1': 'ami-0729d913a335efca7',
             'us-east-2': 'ami-070f4af81c19b41bf',
-            # This AMI is 20210623 as aws does not provide a newer one.
-            'us-west-1': 'ami-0b3c34d643904a734',
+            'us-west-1': 'ami-0365e546798d80d88',
```

Comment:
This region does have the `20220308` version of the AMI. It should be supported.